ai/extract_features.py

```
@@ -316,11 +304,9 @@
     vectorizer.fit(tfidf_texts)
 
     return (
-        _extracted_from_extract_features_73(
-            df, vectorizer, category, n_components, random_state
-        )
+        _extracted_from_extract_features_73(df, vectorizer, category, n_components, random_state)
         if len(df) > 1000000
```

Ruff (0.11.9)

Magic value used in comparison, consider replacing `1000000` with a constant variable

(PLR2004)